lib/util.mjs

```diff
@@ -81,3 +80,17 @@ export function withLogs(execaFn) {
     return execaFn(cmd, args, opts);
   };
 }
+
+/**
+ * Retrieves the package short name by removing the namespace prefix for Google libraries.
+ */
+export function getPackageShortName(packageName) {
+  const packageShortNameRegex = /@google-[^\/]+\//g
+
+  const hasCloudNamespace = packageShortNameRegex.test(packageName)
+  if (!hasCloudNamespace) {
+    throw new Error('Package does not have required namespace prefix.')
```

Comment:
Include the required namespace and the bad package name.

Reply:
Done.